admin/pre_reg/views.py

```diff
@@ -137,10 +129,20 @@ def update_draft(request, draft_pk):
     data = json.loads(request.body)
```

Comment:
Somewhere in here you should be passing request.POST to the form and calling .is_valid(). https://docs.djangoproject.com/en/1.8/topics/forms/#the-view